packages/sdk-metrics/src/Descriptor.ts

```diff
@@ -54,10 +81,10 @@ export function createInstrumentDescriptor(
   };
 }
 
-export function createInstrumentDescriptorWithView(
+export function createDescriptorWithView(
```

Comment:
```suggestion
export function createMetricDescriptorWithView(
```

Reply:
also fixed in fixed in [afde29d](https://github.com/open-telemetry/opentelemetry-js/pull/3520/commits/afde29db015198612d7d11a8bc558b84d1db5247) :slightly_smiling_face: